xapian-applications/omega/date.cc

```diff
@@ -130,9 +132,15 @@ parse_date(const string & date, int *y, int *m, int *d)
 {
     // FIXME: for now only support YYYYMMDD (e.g. 20011119)
     // and don't error check
```

Comment:
The use of stoi can be the fix for the above FIXME as it will check only numbers if the string is encountered then it will throw exception